client/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "Name-Request",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bcgov/namerequest.git"
+  },
+  "version": "0.1.0",
+  "private": true,
+  "license": "Apache License, Version 2.0",
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "test:unit": "vue-cli-service test:unit",
+    "test:e2e": "vue-cli-service test:e2e",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "babel-plugin-transform-decorators": "^6.24.1",
+    "core-js": "^3.3.2",
+    "material-design-icons-iconfont": "^5.0.1",
+    "roboto-fontface": "*",
```

Comment:
There's a BC Gov recommended (mandatory maybe?) font. You can snag it from us here: https://github.com/bcgov/lear/tree/master/coops-ui/src/assets/fonts/BCSans